### DISCUSSION TITLE
PR build reviews for DOM and Javac effort

### DISCUSSION THREAD

**Discussion by mickaelistria** [eclipse-jdtls/eclipse-jdt-core-incubator#23] [Show and tell]
Hi,
I've managed to setup some not-too-intrusive approach to get efficient automated feedback for PRs submitted against dom-based-operations or dom-with-javac branches. This runs a 2nd build enabling those features flags on top of the PR, and report the amount of tests run/errors; so the goal of reducing amount of failures to 0 becomes easier to target.
Any change that 1. doesn't break the "regular" build so gets a ✔ for it and, 2. does *not* increase the amount of test failures/errors for dom-based stuff is most likely welcome for a merge. On the opposite, any change that breaks the regular build or introduce more failures compared to the target branch needs to be fixed.
Here is a screencast of how to get the test count quickly for dom-based:
[Screencast from 2024-02-12 09-51-56.webm](https://github.com/eclipse-jdtls/eclipse-jdt-core-incubator/assets/469058/3ca6a2b3-ff4d-44cc-b858-4335586e7dd4)

cc @datho7561 @robstryker

**Comment by fbricon** [comment]
from your screencast, the test results action should be shown as failed, given the amount of errors/failure > 0

**Reply by mickaelistria** [reply]
The build fails when there is a failure already.
